models/Tool/CustomReport/Adapter/Sql.php

```diff
@@ -191,16 +191,20 @@ protected function getBaseQuery($filters, $fields, $ignoreSelectAndGroupBy = fal
                         case 'lt':
                         case 'gt':
                         case 'eq':
-                            if ($type == 'date') {
-                                $condition[] = $db->quoteIdentifier($filter['property']) . ' BETWEEN ' . $db->quote($value) . ' AND ' . $db->quote($maxValue);
-                                break;
-                            }
-
                             $compMapping = [
                                 'lt' => '<',
                                 'gt' => '>',
                                 'eq' => '='
                             ];
+                            
+                            if ($type == 'date') {
+                                if ($operator == 'eq') {
+                                    $condition[] = $db->quoteIdentifier($filter['property']) . ' BETWEEN FROM_UNIXTIME(' . $db->quote($value) . ') AND FROM_UNIXTIME(' . $db->quote($maxValue) . ')';
```

Comment:
If `$operator` is `eq`shouldn't be the condition `=` ? and not `BETWEEN` ?

Reply:
it was already in the file before my change with the from_unixtime.